wgsl/extract-grammar.py

```diff
@@ -434,8 +434,8 @@ def not_token_only(value):
 subprocess.run(["npm", "install"], cwd=grammar_path, check=True)
 subprocess.run(["npx", "tree-sitter", "generate"],
                cwd=grammar_path, check=True)
-subprocess.run(["npx", "tree-sitter", "build-wasm"],
-               cwd=grammar_path, check=True)
+# subprocess.run(["npx", "tree-sitter", "build-wasm"],
```

Comment:
nit: add a comment saying why this is commented out

Reply:
@kainino0x thanks, done with the new commit